tests/test_snapalgo.py

```diff
@@ -1,11 +1,10 @@
-import unittest
+import unittest.mock
```

Comment:
Why do this? Could explicitly import `MagicMock` instead?

Reply:
It gave me an error in the setup I had at the time to just import unittest but then use unittest.mock.* . This might be a version problem.